tests/network-tests/src/fixtures/membership/UpdateProfileHappyCaseFixture.ts

```diff
@@ -56,8 +55,6 @@ export class UpdateProfileHappyCaseFixture extends BaseQueryNodeFixture {
       expected.externalResources?.map(asMembershipExternalResource) ?? []
     )
     assert.isFalse(Utils.hasDuplicates(metadata.externalResources?.map(({ type }) => type)))
-    assert.equal(metadata.isVerifiedValidator, false)
```

Comment:
Please keep this `assert.equal(metadata.isVerifiedValidator, false)`. Even though it doesn't mean what we thought it did, this value should still be set to`false` when any metadata changes.